File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.js
function solution(a, b, c) {
    var answer = 0;
    if(a!=b && b!=c && a!=c) answer= a+b+c;
    else if(a==b && a!=c) answer=(a+b+c)*(a*a+b*b+c*c);
    else if(b==c && c!=a) answer=(a+b+c)*(a*a+b*b+c*c);
    else if(a==c && a!=b) answer=(a+b+c)*(a*a+b*b+c*c);
    else answer=(a+b+c)*(a*a+b*b+c*c)*(a*a*a+b*b*b+c*c*c);
    return answer;
}

File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.py
def solution(a, b, c):
    answer = 0
    if a!=b and b!=c and a!=c: answer= a+b+c
    elif a==b and a!=c: answer=(a+b+c)*(a*a+b*b+c*c)
    elif b==c and c!=a: answer=(a+b+c)*(a*a+b*b+c*c)
    elif a==c and a!=b: answer=(a+b+c)*(a*a+b*b+c*c)
    else: answer=(a+b+c)*(a*a+b*b+c*c)*(a*a*a+b*b*b+c*c*c)
    return answer